ca/config.go

```diff
@@ -124,10 +124,15 @@ func SigningPolicy(certExpiry time.Duration) *cfconfig.Signing {
 		certExpiry = DefaultNodeCertExpiration
 	}
 
+	notBefore := time.Now().Round(time.Minute).Add(CertDefaultBackdate).UTC()
+	notAfter := time.Now().Round(time.Minute).Add(certExpiry).UTC()
+
 	return &cfconfig.Signing{
 		Default: &cfconfig.SigningProfile{
-			Usage:  []string{"signing", "key encipherment", "server auth", "client auth"},
-			Expiry: certExpiry,
+			Usage:     []string{"signing", "key encipherment", "server auth", "client auth"},
+			Expiry:    certExpiry,
+			NotBefore: notBefore,
+			NotAfter:  notAfter,
```

Comment:
Probably a dumb TLS question, but is it correct to specify both `NotBefore` / `NotAfter` and also `Expiry`? Does cfssl use both of them?